packages/core/test/validate_test.js

```diff
@@ -239,20 +240,23 @@ describe("Validation", () => {
           }
           return errors;
         };
-        const formData = { pass1: "a", pass2: "b" };
+        const formData = { pass1: "a", pass2: "b", numberWithMinimum: 2 };
         const result = validateFormData(formData, schema, validate);
         errors = result.errors;
         errorSchema = result.errorSchema;
       });
 
       it("should return an error list", () => {
-        expect(errors).to.have.length.of(1);
-        expect(errors[0].stack).eql("pass2: passwords don't match.");
+        expect(errors).to.have.length.of(2);
+        expect(errors[0].stack).eql(".numberWithMinimum should be >= 5");
+        expect(errors[1].stack).eql("pass2: passwords don't match.");
```

Comment:
Note how the stack message is subtly different here:
RJSF custom errors use `fieldName: message`
AJV custom errors use `.field.path message`

I am slightly worried that this might be considered a breaking change, as previously our mangling of AJV errors would have changed the stack message to the RJSF format, so anything expecting the stack to remain the same will suffer.

Perhaps I should add an exception to rewrite AJV error stacks to the RJSF stack format?

Reply:
> My recommendation is changing RJSF to use AJV style messages in all cases

I like this approach. I think rjsf is due for a new breaking change soon (there are several PRs that would require breaking changes but are probably needed), so I think we should be fine if we just change this PR to do this.

Reply:
I switched custom errors to use AJV style error messages.

Note that errors on the root now show up with a single dot, as in ". This is a root error".  This appears a bit strange, but is indeed the way AJV shows root errors.